recipes/nionui-tool/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "nionui-tool" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/nion-software/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: cbb89a12b94c48cb28c83e0da8503eaf0c7d4fdcb6720a1d97f1b172f53e51f9
+  - url: https://github.com/nion-software/{{ name }}/releases/download/{{ version }}/LauncherApp.tar.gz  # [osx]
+    sha256: eec013c5bfc49a0178ff872a539280c212edaf3d1d633dc1658f3ec2e5f6f31d  # [osx]
+    folder: "Nion UI Launcher.app"  # [osx]
+  - url: https://github.com/nion-software/{{ name }}/releases/download/{{ version }}/launcher_app.tar.gz  # [linux]
+    sha256: 0eb7c92345b9fb2e583883be327bb482d0df08c05b66376dafd0656d8a5c4c7c  # [linux]
+  - url: https://github.com/nion-software/{{ name }}/releases/download/{{ version }}/LauncherAppWindows.zip  # [win]
+    sha256: 1cfff398e31eefc3622834067c3deb60c23e87c1c1e2bdc458a9ee4428e66bf9  # [win]
+    folder: LauncherAppWindows  # [win]
```

Comment:
Please build from the source code, rather than repackaged binaries: https://github.com/nion-software/nionui-tool/archive/0.4.1.tar.gz

Reply:
That is not an option for us. The build environments on the build system are not suitable for building our project. For instance, there is no sensible way to build a Mac OS application that I can find.

Reply:
We usually do not repackage binaries on conda-forge so that we can ensure that everything is ABI compatible

Reply:
Generally I think that is a great goal - in this case, our application acts more as the host for Python rather than a library of Python. In addition, Python is linked dynamically and we are careful to only pass the simple data types (i.e. nothing is allocated/deallocated across library boundaries). And it is also a goal to have everything built on the conda-forge build environments. Unfortunately, I need a solution which allows me to (1) build applications on Mac OS, presumably with CMake; and (2) integrate the Qt tools present in `pyside2`  like `moc` into the CMake build. I haven't been able to find any other conda-forge projects which build a Mac OS application as part of their build; nor any that use the Qt binary tools in CMake.

Any ideas are welcome!

Reply:
I think what you are trying to do is theoretically possible. I am not entirely clear on all of the parts of the stack that you are talking about.  E.g. I don't know how a Mac App differs from just a tool that is installed. Have you tried asking on the Gitter channel?

Reply:
After a bit of an adventure with CMake and the conda-forge build environment (see https://github.com/conda-forge/conda-forge.github.io/pull/1007) I've got everything running under CMake and building from source.

Please review again -- and a question for you: do all of my exploratory commits get squashed automatically before merging? If not, I would like to squash them before merging this recipe.